README.md

```diff
@@ -0,0 +1,28 @@
+# Vouch Helm Charts
+
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.
+Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+
+Once Helm is set up properly, add the repo as follows:
+
+```console
+#TODO
```

Comment:
reminder to fix this

Reply:
Need to work with @bnfinet on maybe setting up a github pages place to publish the packages.

Reply:
Done now.